lms/resources/_js_config/__init__.py

```diff
@@ -587,4 +587,11 @@ def _get_lti_launch_debug_values(self):
         }
 
     def _generate_annotation_metadata(self, assignment):
-        return {"lms": {"assignment_id": assignment.resource_link_id}}
+        return {
+            "lms": {
+                "assignment": {
+                    "guid": self._application_instance.tool_consumer_instance_guid,
```

Comment:
I reckon `guid` belongs to the top level structure. 

It's not an assignments specific value but an institution-wide one, we just happen to use it for  querying assignments now.

Reply:
https://github.com/hypothesis/lms/pull/5790